src/components/WearablePreview/WearablePreview.stories.tsx

```diff
@@ -28,18 +28,24 @@ const getRandomHex = () => {
   return '#' + Math.random().toString(16).slice(2, 8)
 }
 
+const getRandomProfile = () => {
+  return 'default' + ((Math.random() * 50) | 0)
```

Comment:
is it required to use the `| 0`?

Reply:
It is there in order to remove the decimals. The expected result should be something like `default12`, `default23`, `default50`, etc. Without the `| 0`, the `Math.random() * 50` would result in floating numbers, and I only want the integer part.

Reply:
example:

```
Math.random() * 50
27.101958038036145
Math.random() * 50
19.66821697747213
Math.random() * 50
15.909330523962307
Math.random() * 50 | 0
49
Math.random() * 50 | 0
2
Math.random() * 50 | 0
26
```

Reply:
got it! 🚀